app/models/vmdb_database_setting.rb

```diff
@@ -20,25 +20,13 @@ def vmdb_database
 
   delegate :id, :to => :vmdb_database, :prefix => true
 
-  def minimum_value
-    min_val || ''
-  end
-
-  def maximum_value
-    max_val || ''
-  end
-
-  def value
-    setting || ''
-  end
+  alias_attribute :minimum_value, :min_val
```

Comment:
This may have been too much.
I went into the screens and they seemed to work the same, but you are right, I do not know.